### ISSUE TITLE
[HOLD #9660] [$250] Copy link option is shown for app links - Reported by @thesahindia

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#9975]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:9
1. Go to settings > About > app download links
2. Right click on a link

## Expected Result:
Copy link option shouldn't be shown

## Actual Result:
We have an option to copy link

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.85-0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
<img width="372" alt="Screenshot 2022-07-03 at 4 34 56 PM" src="https://user-images.githubusercontent.com/43995119/179663570-dd37603d-924d-4e06-a81c-47252a06597e.png">

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @thesahindia
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1656846795990329

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by parasharrajat** [comment]
I think the whole menu should not be shown. Good catch @thesahindia

**Comment by laurenreidexpensify** [comment]
Just so I understand better, can you explain a bit more why you think this is a problem @thesahindia @parasharrajat ? I'm thinking that maybe we would want to give users the option to cut the URL and be able to share it to help us with virality?

**Comment by parasharrajat** [comment]
There are two things here: 
1. First, the copy link option on the menu should not be present for links. This is not meant for these links. 
2. Why is this menu popping up for app menu items? I am not sure if this was done intentionally but as per my knowledge, it was not the case a few months back. 

--- 

OK, I can confirm 2 is intentional as per the https://github.com/Expensify/App/pull/9047. But 1 is still an issue.

**Comment by NikkiWines** [comment]
Able to reproduce on staging and looks fine for a contributor to pick up. Adding `External`

**Comment by NikkiWines** [comment]
Oops sorry @stephanieelliott didn't mean to unassign you

**Comment by stephanieelliott** [comment]
Posted to Upwork: https://www.upwork.com/jobs/~0115c0cba0d60a784f

**Comment by stephanieelliott** [comment]
Hey @thesahindia, as the reporter of this issue you have first dibs on the job! Please post a proposal if you have one.

**Comment by errahsoufiane** [comment]
### **Proposal**
Option 1: whole menu should not be shown
https://github.com/Expensify/App/blob/bfbdd4ed666ea00a0e6d5113369777b5b49f8e41/src/pages/settings/AppDownloadLinks.js#L88
remove this line

Option 2: `Copy link` should not be shown. `Copy URL to clipboard` can still be shown
https://github.com/Expensify/App/blob/bfbdd4ed666ea00a0e6d5113369777b5b49f8e41/src/pages/home/report/ContextMenu/ContextMenuActions.js#L124
replace this line with:
```
        shouldShow: type => type !== CONTEXT_MENU_TYPES.LINK,
```
<img width="375" alt="app download links" src="https://user-images.githubusercontent.com/108141252/181345062-13ce56b2-398d-47ad-8a4d-2ac7340147f4.png">

**Comment by parasharrajat** [comment]
This is kind of a duplicate as the solution we are working for https://github.com/Expensify/App/issues/9660 in PR https://github.com/Expensify/App/pull/9955 will fix this.

**Comment by Santhosh-Sellavel** [comment]
Thanks, @parasharrajat!

@flodnv 
Let's remove help wanted here and wait for #9660's PR to be merged. 
Once PR is merged, lets verify and close this one until then let's add `On hold for 9660` to issue title.

**Comment by flodnv** [comment]
Agreed & done, thanks!

**Comment by stephanieelliott** [comment]
No update, moving this to monthly while it's on hold!

**Comment by stephanieelliott** [comment]
https://github.com/Expensify/App/issues/9660's PR was merged, just tested and this issue is no longer occurring. Closing this one out!
